.github/actions/setup/action.yml

```diff
@@ -42,6 +42,35 @@ runs:
         with:
           distribution: 'temurin' # use same JDK distro as in Trino docker images
           java-version: ${{ inputs.java-version }}
+      - name: Get NodeJS version from parent pom
+        shell: bash
+        run: |
+          node_version=$(grep -m 1 '<dep.frontend-node.version>' "pom.xml" | sed -n 's/.*<dep.frontend-node.version>\(.*\)<\/dep.frontend-node.version>.*/\1/p')
```

Comment:
Why not use maven to get the property value? Same as we do to get the project version in a few other places

Reply:
I'd need to install project first to get a clean output

Reply:
Wdym by clean output?